api/routes/v1/sic_lookup.py

```diff
@@ -72,4 +82,11 @@ async def sic_lookup(
             detail=f"No SIC code found for description: {description}",
         )
 
+    duration_ms = int((time.perf_counter() - start_time) * 1000)
+    logger.info(
```

Comment:
Could you add the code in the case a match was found?

For correlation we could truncate the org desc and append the timestamp the request was received?